src/Commands/Modules/Utility/MathCommand.cs

```diff
@@ -0,0 +1,66 @@
+﻿namespace BrackeysBot.Commands
+{
+    using System.Drawing;
+    using System.Drawing.Drawing2D;
+    using System.Drawing.Imaging;
+    using System.IO;
+    using System.Threading.Tasks;
+    using CSharpMath.SkiaSharp;
+    using Discord.Commands;
+    using SkiaSharp;
+
+    public partial class UtilityModule : BrackeysBotModule
+    {
+        [Command("math"), Alias("maths", "eq", "equation", "tex", "latex")]
+        [Summary("Renders LaTeX input to an image, and attaches the result.")]
+        public async Task MathCommandAsync([Summary("The LaTeX input."), Remainder]
+                                           string input)
```

Comment:
Can attributes and parameter can go on a single line? Even if its a complete newline.